yt_dlp/extractor/tencent.py

```diff
@@ -63,7 +70,9 @@ def _get_video_api_response(self, video_url, video_id, series_id, subtitle_forma
 
     def _extract_video_formats_and_subtitles(self, api_response, video_id):
         video_response = api_response['vl']['vi'][0]
-        video_width, video_height = video_response.get('vw'), video_response.get('vh')
+        has_drm, video_width, video_height, video_format = video_response.get('drm') == 1, video_response.get('vw'), video_response.get('vh'), video_response.get('br')
```

Comment:
Avoid declaring variables only used once

https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#inline-values

Reply:
Ok, I'll fix it when I wake up.